Resources/config/twig.xml

```diff
@@ -8,5 +8,9 @@
         <service id="sonata.user.twig.global" class="Sonata\UserBundle\Twig\GlobalVariables" >
             <argument type="service" id="service_container" />
         </service>
+
+        <service id="sonata.user.twig.extensions" class="Sonata\UserBundle\Twig\SecurityExtensions">
```

Comment:
I don't know. If the extension filter/methods are available even the service it private, then yes.